client/blocks/product-selector/index.jsx

```diff
@@ -355,12 +355,10 @@ export class ProductSelector extends Component {
 		return (
 			<ProductCardAction
 				onClick={ this.handleCheckoutForProduct( productObject ) }
-				intro={ this.getIntervalDiscount( selectedProductSlug ) }
-				label={ translate( 'Get %(productName)s', {
-					args: {
-						productName: this.getActionButtonName( product, productObject.product_slug ),
-					},
-				} ) }
+				intro={ this.props.selectedSiteSlug && this.getIntervalDiscount( selectedProductSlug ) }
+				label={
+					translate( 'Get ' ) + this.getActionButtonName( product, productObject.product_slug )
```

Comment:
This change doesn't look correct ("Get %(productName)s" should be the correct way to translate the phrase, so the translator gets context about what the "Get" refers to).

I think to make the product name itself non-translatable (if that's the direction that it makes sense to go in) you would need to modify the product names in [getJetpackProductsShortNames()](https://github.com/Automattic/wp-calypso/blob/4537033c6794449ce0a10b69d72daf5fa658d553/client/lib/products-values/constants.js#L78]) (and I guess `getJetpackProductsDisplayNames()` underneath it too).

Reply:
thanks @DavidRothstein, reverted. Let;s handle it in a separate PR.